packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -33,15 +40,9 @@ export default {
       value: `<img slot='media-back' class="object-cover h-full w-full" src='./placeholders/images/generic.jpg' alt='Generic'/>`
```

Comment:
Please use for the default the primary variant.
![image](https://github.com/user-attachments/assets/fb3d7bdf-4d78-4683-af56-fb50465ae8e5)

Reply:
Hey @yoezlem, a issue was opened for it: https://github.com/orgs/solid-design-system/projects/1/views/1?pane=issue&itemId=76163370.
Once the issue is fixed, the default will be automatically updated.